QuizBowlDiscordScoreTracker/Bot.cs

```diff
@@ -388,5 +388,9 @@ private async Task UnmuteReaderAfterDelay(IVoiceChannel voiceChannel, IGuildUser
                 await reader.ModifyAsync(properties => properties.Mute = false);
             }
         }
+
+        public static string groupFromChannel(ITextChannel channel) {
```

Comment:
Nitpick: method names should be capitalized, like GroupFromChannel, to be consistent with the rest of the codebase

Reply:
fixed in 848e026